awsbatch-cli/requirements.txt

```diff
@@ -0,0 +1,3 @@
+boto3>=1.16.14
+tabulate>=0.8.2,<0.8.10
+configparser>=3.5.0
```

Comment:
do we still need this with python > 3.6?

Reply:
Removed